flask_apispec/extension.py

```diff
@@ -67,11 +67,21 @@ def swagger_json(self):
     def swagger_ui(self):
         return flask.render_template('swagger-ui.html')
 
-    def register(self, target, endpoint=None, blueprint=None):
+    def register(self, target, endpoint=None, blueprint=None, **kwargs):
```

Comment:
Since this is a public method, I'd rather see `resource_class_args` and `resource_class_kwargs` explicitly declared in the function signature for readability and better static analysis.